cryptographic_estimators/estimation_renderer.py

```diff
@@ -20,7 +20,7 @@
 from .helper import concat_all_tables, round_or_truncate
 from copy import deepcopy
 from prettytable import PrettyTable
-from sage.all import RR
+from numpy import float64
```

Comment:
why numpy? I just grepped over the repo and thats the only place where we include numpy

Reply:
@FloydZ Happy to hear that!! :rocket: And thank you!

About numpy, yes, that's tricky. The reason is because we need to replace the check for the rows, from being in sage RR, to being of the type `float` or `float64`, as seen here: 
https://github.com/Crypto-TII/CryptographicEstimators/blob/94d574f404f4defdb5a62fa34c971657ce97637a/cryptographic_estimators/estimation_renderer.py#L124-L126

Otherwise, the rounding of the rows doesn't work. The reason is probably because of previous calculations using SciPy or SymPy, as both of them use numpy under the hood (if I remember correctly).